internal/collectors/github/organization_collector.go

```diff
@@ -89,10 +99,16 @@ func (c *organizationCollector) collectExtraData(org *ghcollected.ExtendedOrg) g
 		log.Printf("failed to collect webhooks data for %s, %s", org.Name(), err)
 	}
 
+	enterpriseVisibilityChangePolicyDisabled, err := c.collectEnterpriseVisibilityChangePolicyDisabled()
```

Comment:
only valid when the organization is part of the enterprise otherwise should be nil